lighthouse-core/report/html/report-template.html

```diff
@@ -36,17 +36,30 @@
   </script>
   <script>window.__LIGHTHOUSE_JSON__ = %%LIGHTHOUSE_JSON%%;</script>
   <script>
+    let pollyfillsLoaded = Promise.resolve();
+    if (!("open" in document.createElement("details"))) {
+      pollyfillsLoaded = new Promise((resolve, reject) => {
+        const script = document.createElement('script');
+        script.src = 'https://unpkg.com/details-element-polyfill@2.0.4/dist/details-element-polyfill.js';
+        script.onload = resolve;
+        script.onerror = reject;
```

Comment:
in this case, i think it'd be an uncaught rejection. Perhaps we set onerror to `resolve` as well? Better to try to render something than give up completely in this case.

Reply:
wasn't 100% sure about that but i'm ok with that!